src/ontology/mondo-ingest.Makefile

```diff
@@ -113,7 +113,6 @@ $(COMPONENTSDIR)/doid.owl: $(TMPDIR)/doid_relevant_signature.txt | component-dow
 		remove -T $(TMPDIR)/doid_relevant_signature.txt --select complement --select "classes individuals" --trim false \
 		query \
 			--update ../sparql/fix_omimps.ru \
-			--update ../sparql/fix_hgnc_mappings.ru \
```

Comment:
## Suggestion: file rename
If indeed I'm correct and `fix_hgnc_mappings.ru` only applies to OMIM, why don't we prepend `omim_` to the file name for disambiguation?

Reply:
That's a good question. I'll look into that.

Reply:
Is this still an open action item, to look into whether the "fix" hgnc mappings is still necessary?

Reply:
yep

Reply:
@matentzn You were correct. I ran with and without this SPARQL update, and there was no difference. I've deleted the file and its references. Feel free to merge!